src/collection-browser.ts

```diff
@@ -245,70 +250,105 @@ export class CollectionBrowser
   }
 
   render() {
+    this.doesHaveSearchTermOrResult();
```

Comment:
q: is `this.doesHaveSearchTermOrResult();` needs to be called every time it is rendered? or could be just on first load?

Reply:
So we wants to check is is already loading and also the total results. so I need to call it after the firstUpdated() and Updated() function because later it will get correct results count to show placeholders. that's why I can't call it with first load.